util/llpcInternal.cpp

```diff
@@ -616,18 +624,22 @@ Result VerifySpirvBinary(
     };
 #undef _SPIRV_OP
 
-    const uint32_t* pCode = reinterpret_cast<const uint32_t*>(pSpvBin->pCode);
-    const uint32_t* pEnd  = pCode + pSpvBin->codeSize / sizeof(uint32_t);
-
+    const char* pCodeBegin = static_cast<const char*>(pSpvBin->pCode);
+    const char* pCodeEnd = pCodeBegin + pSpvBin->codeSize;
     // Skip SPIR-V header
-    const uint32_t* pCodePos = pCode + sizeof(SpirvHeader) / sizeof(uint32_t);
+    const char* pCodePos = pCodeBegin + sizeof(SpirvHeader);
+    constexpr size_t wordSize = sizeof(uint32_t);
 
-    while (pCodePos < pEnd)
+    while (pCodePos < pCodeEnd)
     {
-        Op opCode = static_cast<Op>(pCodePos[0] & OpCodeMask);
-        uint32_t wordCount = (pCodePos[0] >> WordCountShift);
+        uint32_t currWord;
+        FromRawBytes(pCodePos, pCodeEnd, currWord);
```

Comment:
please check whether it generated additional call in generated code in release build.
I don't want this change causes any compiler performance drop.

Reply:
All the calls are gone on my machine.

Reply:
got it. thanks!